pom.xml

```diff
@@ -14,6 +14,11 @@
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
 
+  <properties>
+    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
+    <findbugs-maven-plugin.failOnError>true</findbugs-maven-plugin.failOnError>
```

Comment:
@stephenc please check maillist i sent my thought about this variables. Maybe you can propose something better.